src/Controllers/ModuleSuiteLocator.php

```diff
@@ -190,4 +196,30 @@ protected function loadSuiteConfiguration($suite, Module $module)
             'settings' => $resolvedConfig,
         ];
     }
+
+    private function setI18nLocale(array $config): void
+    {
+        $path = $config['type']['settings']['paths'][0] ?? '';
+        $packagistName = '';
+        if (preg_match('#^vendor/([^/]+/[^/]+)/.+#', $path, $matches)) {
```

Comment:
Locally the path for me when running admin behat is `/var/www/html/vendor/silverstripe/admin/tests/behat/features` which doesn't match this regex and therefore the locale doesn't get set.

Reply:
Updated